source/_integrations/glances.markdown

```diff
@@ -39,6 +39,12 @@ If this doesn't work, try changing the `3` to `2`, if you don't have the latest
 
 For details about auto-starting `glances`, please refer to [Start Glances through Systemd](https://github.com/nicolargo/glances/wiki/Start-Glances-through-Systemd).  
 
+<div class='note warning'>
```

Comment:
I think we should move the warning to right under the "Setup" header.

Also, I think we should consider removing the text about testing the API and using different versions. Users should just start the service and then try to set up the integration in Home Assistant. We'll now try both version 3 and 2 automatically.

Reply:
@engrbm87, are you good with the changes to go into this beta?

Reply:
I believe Martin had some suggestions, but unfortunately I don't have time at the moment to address them.
If you can include these suggestions then we should be good to go.

Reply:
ok, will do. Thank you for the feedback!